File: 0125-valid-palindrome/0125-valid-palindrome.cs
public class Solution {
    public bool IsPalindrome(string s) {
        
        var query = s.Where(c => Char.IsLetterOrDigit(c)).Select(c => Char.ToLower(c));
        var arr = query.ToArray(); 
        Array.Reverse(arr); 
        
        return new string(query.ToArray()) == new string(arr); 

    }
}

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        
        vector<char> arr1, arr2;
        
        for(int i=0; i<s.size(); i++) {
            
            if(isalnum(s[i])) {
                
                arr1.push_back(tolower(s[i]));
            }
            
            
        }
        arr2 = arr1;
        reverse(arr1.begin(), arr1.end());
        
        return arr1 == arr2;
        
    }
};